Models/Ticket.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace GeekHub.Models
+{
+    public class Ticket
+    {
+        public Guid Id { get; set; }
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public TicketType TicketType { get; set; }
+        public TicketStatus TicketStatus { get; set; }
+        public Guid ProjectId { get; set; }
+        public Project Project { get; set; }
+        public Guid? ReporterId { get; set; }
+        public virtual ApplicationUser Reporter { get; set; }
```

Comment:
Why have both the `Reporter` and `ReporterId`?

Reply:
Because when creating the ticket it is easier to set the Id